homeassistant/components/p1_monitor/config_flow.py

```diff
@@ -59,6 +59,7 @@ async def async_step_user(
                     vol.Required(CONF_HOST): TextSelector(),
                     vol.Required(CONF_PORT, default=80): NumberSelector(
                         NumberSelectorConfig(
+                            as_int=True,
```

Comment:
You can't use that yet... and it won't make it into a patch release.
Please use vol.All

Reply:
Alright I will temporarily change it to `vol.All` now, will you include the P1 Monitor in your PR?

Reply:
Yes I will update if/when this one is merged